src/routes/transactions/tests/preview.rs

```diff
@@ -0,0 +1,141 @@
+use std::time::Duration;
+
+use mockall::predicate::eq;
+use rocket::http::{ContentType, Header, Status};
+use rocket::local::asynchronous::Client;
+use serde_json::json;
+
+use crate::common::models::backend::chains::{
+    BlockExplorerUriTemplate, ChainInfo, GasPrice, NativeCurrency, RpcAuthentication, RpcUri, Theme,
+};
+use crate::config::contract_info_request_timeout;
+use crate::providers::info::*;
+use crate::routes::contracts::models::DataDecoderRequest;
+use crate::routes::transactions::handlers::preview::TransactionPreview;
+use crate::routes::transactions::tests::PREVIEW_RESPONSE;
+use crate::tests::main::setup_rocket;
+use crate::utils::errors::ApiResult;
+use crate::utils::http_client::{MockHttpClient, Request, Response};
+
+fn get_chain_info() -> ChainInfo {
+    ChainInfo {
+        recommended_master_copy_version: "1.1.1".to_string(),
+        transaction_service: "https://safe-transaction.mainnet.gnosis.io".to_string(),
+        vpc_transaction_service: "http://mainnet-safe-transaction-web.safe.svc.cluster.local"
```

Comment:
could we not expose internal information here? :D

Reply:
Addressed in [3ddb403](https://github.com/safe-global/safe-client-gateway/pull/924/commits/3ddb4035096ab6434cac56ab9838a9a089bf711b)